docs/getting-started/build-a-sample-app.md

```diff
@@ -99,19 +125,20 @@ Finally, add an `index.html` file in the root directory of your app:
 </html>
 ```
 
-## Run your app
+## View your web applcation
 
-If you haven't already, install [Trunk](https://github.com/thedodd/trunk):
+Run the following command to build your application, launch a local web hosting environment, and launch a web browser to view your application.
 
 ```bash
-cargo install --locked trunk
-cargo install wasm-bindgen-cli
+trunk serve
 ```
 
-Now all you have to do is run the following:
+This will start a development server which continually updates the app every time you change something in your web application.
 
-```bash
-trunk serve
-```
 
-This will start a development server which continually updates the app every time you change something.
+## Congratulations
+
+You have now successfully setup your Yew development environment, and built your first web application.
+
+Experiment with this application and review the [examples](./examples.md) to further your learning.
```

Comment:
Not really a problem otherwise.
```suggestion
Feel free to experiment with this application; you might find the [examples](./examples.md) useful.
```